application/jira/v1/config/tasks.json

```diff
@@ -0,0 +1,1181 @@
+                "custom-issue-type"
+              ],
+              "instillFormat": "object",
+              "type": "object"
+            }
+          ]
+        },
+        "status": {
+          "title": "Status",
+          "description": "The status of the issue, can be: `To Do`, `In Progress`, `Done`",
+          "instillShortDescription": "The status of the issue",
+          "instillUIOrder": 4,
+          "instillFormat": "string",
+          "instillAcceptFormats": [
+            "string"
+          ],
+          "instillUpstreamTypes": [
+            "value",
+            "reference",
+            "template"
+          ],
+          "type": "string"
+        },
+        "summary": {
+          "title": "Summary",
+          "description": "The summary of the issue",
+          "instillShortDescription": "The summary of the issue",
+          "instillUIOrder": 2,
+          "instillFormat": "string",
+          "instillAcceptFormats": [
+            "string"
+          ],
+          "instillUpstreamTypes": [
+            "value",
+            "reference",
+            "template"
+          ],
+          "type": "string"
+        },
+        "description": {
+          "title": "Description",
+          "description": "The description of the issue",
+          "instillShortDescription": "The description of the issue",
+          "instillUIOrder": 3,
+          "instillFormat": "string",
+          "instillAcceptFormats": [
+            "string"
+          ],
+          "instillUpstreamTypes": [
+            "value",
+            "reference",
+            "template"
+          ],
+          "type": "string"
+        },
+        "assignee": {
+          "title": "Assignee",
+          "description": "The assignee of the issue",
+          "instillShortDescription": "The assignee of the issue",
+          "instillUIOrder": 5,
+          "instillFormat": "string",
+          "instillAcceptFormats": [
+            "string"
+          ],
+          "instillUpstreamTypes": [
+            "value",
+            "reference",
+            "template"
+          ],
+          "type": "string"
+        }
+      },
+      "required": [
+        "project-key",
+        "issue-type",
+        "status"
+      ],
+      "title": "Input",
+      "type": "object"
+    },
+    "output": {
+      "description": "Create an issue in Jira",
+      "instillUIOrder": 0,
+      "$ref": "#/$defs/issue",
+      "title": "Output",
+      "type": "object"
+    }
+  },
+  "TASK_UPDATE_ISSUE": {
+    "description": "Update an issue in Jira",
+    "instillShortDescription": "Update an issue in Jira",
+    "input": {
+      "description": "Update an issue in Jira",
+      "instillUIOrder": 0,
+      "instillEditOnNodeFields": [
+        "issue-key",
+        "update"
+      ],
+      "properties": {
+        "issue-key": {
+          "$ref": "#/$defs/issue/properties/key",
+          "title": "Issue Key",
+          "instillUIOrder": 0
+        },
+        "update": {
```

Comment:
`"update": {`

In this whole design, I am not so sure if Console can do it.

Reply:
Or, maybe we can make update as json.
And, the users have to separate the action in the different component given 1 component execute 1 task now.

Reply:
> make update as json

However, this would lack validation and may make the debugging process difficult for the users, as the front end does not show error message details now.

> And, the users have to separate the action in the different component given 1 component execute 1 task now.

Does it mean that users need to use `iterator` to extract each item in the array and execute 1 at a time?